### PR TITLE
[#1541] Add Xtext-based projects to the Xtext Community Website.

### PR DESCRIPTION
**Pull request by miklossy**
Signed-off-by: Tamas Miklossy <miklossy@itemis.de>

### CODE REVIEW

File: xtext-website/community.html [hunk 1]
```diff
@@ -1094,6 +1094,14 @@ <h2>What others have built with Xtext</h2>
 						<td>Lorenzo Bettini</td>
 					</tr>
 
+					<tr>
+						<td><a href="https://github.com/Yakindu/solidity-ide">YAKINDU Solidity IDE</a></td>
+						<td>The free to use, open source YAKINDU Solidity IDE provides an Xtext-based integrated development environment for ethereum / solidity based smart contracts. It can be installed in different environments, such as Atom, Eclipse, Theia and VsCode. <a href="https://solidity-ide.itemis.de/">Online editor</a> and <a href="https://yakindu.github.io/solidity-ide/">documentation</a>.</td>
```
Comment (kthoms) [inline xtext-website/community.html:1099]: Please fix casing
ethereum -> Ethereum
solidity -> Solidity
VsCode -> VSCode

**Comment by miklossy** [comment]
Thanks for the feedback @kthoms ! I updated the PR as requested.

File: xtext-website/community.html [hunk 2]
```diff
@@ -1094,6 +1094,14 @@ <h2>What others have built with Xtext</h2>
 						<td>Lorenzo Bettini</td>
 					</tr>
 
+					<tr>
+						<td><a href="https://github.com/Yakindu/solidity-ide">YAKINDU Solidity IDE</a></td>
+						<td>The free to use, open source YAKINDU Solidity IDE provides an Xtext-based integrated development environment for Ethereum / Solidity-based smart contracts. It can be installed in different environments, such as Atom, Eclipse, Theia and VSCode. <a href="https://solidity-ide.itemis.de/">Online editor</a> and <a href="https://yakindu.github.io/solidity-ide/">documentation</a>.</td>
```
Comment (cdietrich) [inline xtext-website/community.html:1099]: the sentence ending with documentation makes no sense

Comment (miklossy) [reply xtext-website/community.html:1099]: I took that from the description of the Balzac project:
screenshot
I think it is not so bad.

**Pull request merged**